config/boot.rb

```diff
@@ -37,3 +37,7 @@
 require_relative '../app/models/batch_script'
 require_relative '../app/models/job'
 require_relative '../app/models/job_step'
+
+unless FlightScheduler.env.test?
+  FlightScheduler.app.init_periodic_processor
```

Comment:
:panda_face: This can probably be launched in the `config.ru` to remove the conditional logic. That way in the weird case that `falcon` is started in a `test` environment, it still starts. I assume it is here to prevent it running in `rspec`?

Reply:
Well to give us control over whether we want to run it in the specs.